smelly_python/code_smell.py

```diff
@@ -0,0 +1,28 @@
+from multiprocessing.dummy import Array
+
+
+class Code_Smell(object): 
+    def __init__(self, data):
+        self.type = data['type']
+        self.module = data['module']
+        self.obj = data['obj']
+        self.line = data['line']
+        self.column = data['column']
+        self.endLine = data['endLine']
+        self.path = data['path']
+        self.symbol = data['symbol']
+        self.message = data['message']
+        self.message_id = data['message-id']
+
+    def __repr__(self) -> str:
+        return f"{self.type} in {self.module} on line {self.line} at {self.column} with reason: {self.message}"
```

Comment:
Let's use single quotes everywhere we can

Reply:
ack